src/main/kotlin/blackjack/domain/Deck.kt

```diff
@@ -0,0 +1,29 @@
+package blackjack.domain
+
+class Deck(val cards: Cards) {
+
+    fun getCard(): Card {
+        return cards.removeAt(0)
+    }
+
+    fun getInitGameCards(): Cards {
+        val cards = Cards(mutableListOf())
+        repeat(2) {
+            cards.add(getCard())
+        }
+
+        return cards
+    }
+
+    companion object {
+        fun newDeck(): Deck {
+            val cards = mutableListOf<Card>()
+            for (i in 0 until Rank.values().size * Suit.values().size) {
+                cards.add(Card.valueOf(i))
+            }
+
+            cards.shuffle()
+            return Deck(Cards(cards))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            val cards = Suit.values().flatMap { suit ->
                Rank.values().map { rank -> Card(suit, rank) }
            }
            return Deck(Cards(cards.shuffled().toMutableList()))
```